packages/qwik/src/core/util/string.ts

```diff
@@ -57,6 +57,11 @@ export const packUint8Array = (bytes: Uint8Array) => {
       code += String.fromCharCode(0xfffd, 0xffff);
       continue;
     }
+    // extra comaction againt the strinfigy of the control characters
+    if (c <= 31) {
+      code += String.fromCharCode(0xfffd, c + 0xf200);
```

Comment:
I just realized that because we don't limit ourselves to 0xfffd f..., you can just add 32 and it will be stringified smaller.